### PR TITLE
xar: add page

### PR DESCRIPTION
**Pull request by johan456789**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

**Comment by agnivade** [comment]
You could have added both the files in a single PR itself. :)

File: pages/osx/xar.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# xar
+
+> Manage .xar archives.
+
+- Create a xar archive of all files in {{directory}}:
```
Comment (agnivade) [inline pages/osx/xar.md]: "in a given directory"

File: pages/osx/xar.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+`xar -cf sample.xar {{directory}}`
```
Comment (agnivade) [inline pages/osx/xar.md]: path/to/directory

File: pages/osx/xar.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+- List the contents of the xar archive sample.xar:
```
Comment (agnivade) [inline pages/osx/xar.md]: List the contents of a given xar archive.

File: pages/osx/xar.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+`xar -tf sample.xar`
```
Comment (agnivade) [inline pages/osx/xar.md]: Let's name it to `{{file.xar}}` for consistency with other pages.

File: pages/osx/xar.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+
+- Extract the contents of sample.xar to the current working directory:
+
+`xar -xf sample.xar`
```
Comment (agnivade) [inline pages/osx/xar.md]: `{{file.xar}}`

File: pages/osx/xar.md [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+
+- Create a xar archive of all files in {{directory}}:
+
+`xar -cf sample.xar {{directory}}`
+
+- List the contents of the xar archive sample.xar:
+
+`xar -tf sample.xar`
+
+- Extract the contents of sample.xar to the current working directory:
```
Comment (agnivade) [inline pages/osx/xar.md]: Extract the contents of a xar archive ..

**Comment by sbrl** [comment]
Erm hang on. There's a duplicate PR for this over in #2159 I've just reviewed? Not sure what's up here :confused:  :question:

**Comment by agnivade** [comment]
That's the `linux/` part of the same file.

**Comment by sbrl** [comment]
Ah ok. Got confused for a moment there :P

**Comment by agnivade** [comment]
@johan456789 - The other PR got merged. You just need to copy the same contents on this page and update the PR.

**Pull request closed without merging**